bot/resources/tags/windows-path.md

```diff
@@ -1,30 +1,17 @@
 **PATH on Windows**
 
-If you have installed Python but you forgot to check the *Add Python to PATH* option during the installation you may still be able to access your installation with ease.
+If you have installed Python but forgot to check the *Add Python to PATH* option during the installation, you may still be able to access your installation with ease.
 
-If you did not uncheck the option to install the Python launcher then you will find a `py` command on your system. If you want to be able to open your Python installation by running `python` then your best option is to re-install Python.
+If you did not uncheck the option to install the Python launcher, then you'll instead have a `py` command which can be used in the same way. If you want to be able to access your Python installation via the `python` command, then your best option is to re-install Python (remembering to tick the `add to PATH` checkbox).
```

Comment:
First we say "Check *Add Python to PATH*" but then say "if you did not uncheck"?

Reply:
The `uncheck` is referring to a separate setting (installing the Python launcher)

Reply:
Huh, I don't even know the difference haha

Reply:
Perhaps we should have the option in asterisks to better highlight it?

Reply:
Yeah we can put italic around *Python Installer*?

Reply:
The installer has the setting as `py launcher` so will reword to this and add the asterisks

Reply:
This has been done in [093fdaa](https://github.com/python-discord/bot/pull/1836/commits/093fdaa2b8c89b770d7ff2b7889077dd9e21ad2d)